### DISCUSSION TITLE
ScriptReference/MonoBehaviour.StopCoroutine

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#556] [Notes]
# ScriptReference/MonoBehaviour.StopCoroutine

https://docs.unity3d.com/ScriptReference/MonoBehaviour.StopCoroutine.html

**Comment by tbarusseau** [comment]
Note that `StopCoroutine` throws a `NullReferenceException` if the routine is null (which isn't documented).
